Utils/github_workflow_scripts/github_workflow_scripts_tests/run_docs_review_test.py

```diff
@@ -24,6 +27,7 @@ def test_file_name_includes_apostrophe(self, mocker, capsys):
         mocker.patch('Utils.github_workflow_scripts.run_docs_review.parse_changed_files_names', return_value=args)
 
         result = run_docs_review()
-        captured = capsys.readouterr()
-        assert sdk_docs_reviewer_starting_string in captured.out
+        # When demisto-sdk 1.14.1 is released, it's safe to remove the capsys check
+        assert sdk_docs_reviewer_starting_string in flatten_call_args(logger_info.call_args_list) or \
+            sdk_docs_reviewer_starting_string in capsys.readouterr().out
```

Comment:
Not sure this capsys check is still needed

Reply:
Because 1.14.0 still writes to output, so if I delete it will fail currently.
When demisto-sdk will be released, we can delete the capsys capture